lib/brains/tests/test_management.py

```diff
@@ -0,0 +1,48 @@
+from django.core.management.base import CommandError
+
+from nose.tools import eq_, ok_, raises
+
+from lib.brains.management.commands import braintree_config as config
+from lib.brains.tests.base import BraintreeTest
+
+
+class TestManagement(BraintreeTest):
+
+    @raises(CommandError)
+    def test_bad(self):
+        config.import_file('foo.bar')
+
+    def test_good(self):
+        ok_(config.import_file('solitude.settings.products'))
+
+    def test_created(self):
+        seller = config.get_or_create_seller('uuid:fxa')
```

Comment:
could this be more realistic like ``config.get_or_create_seller('mozilla-concrete')`` ?